src/cli/cli.cpp

```diff
@@ -4570,6 +4570,7 @@ template <> otError Interpreter::Process<Cmd("neighbor")>(Arg aArgs[])
                 OutputFormat("| %3lu ", ToUlong(neighborInfo.mAge));
                 OutputFormat("| %8d ", neighborInfo.mAverageRssi);
                 OutputFormat("| %9d ", neighborInfo.mLastRssi);
+                OutputFormat("| %5u ", neighborInfo.mLinkQualityIn);
```

Comment:
Can you also please update the documentation exmaple in the `cli/REAME.md` here:

https://github.com/openthread/openthread/blob/4d4deabe8f4a7c5e872dc78a1ac1c0b59bd9fb27/src/cli/README.md#neighbor-table

https://github.com/openthread/openthread/blob/4d4deabe8f4a7c5e872dc78a1ac1c0b59bd9fb27/src/cli/README.md?plain=1#L2885-L2898

Reply:
Done with 6d78fad37. It was on my radar as well.
Just wanted to make sure the Column Name and Position is okay before updating the README sample output 😅.